Structure_Engine/Modify/Flip.cs

```diff
@@ -45,6 +52,9 @@ public static Bar Flip(this Bar bar)
 
         /***************************************************/
 
+        [Description("Flips the location curve of the Edge, i.e. the start becomes the end and vice versa.")]
```

Comment:
StartNode becomes the EndNode

Reply:
This is relating to a general curve, not a bar, so nodes involved (and no explicit properties, as they vary with curve type). Think I will leave unchanged?

Reply:
agreed